packages/patternfly-4/react-charts/src/components/ChartUtils/chart-legend.ts

```diff
@@ -60,15 +52,83 @@ export const getLegendDimensions = ({
 
 // Returns x coordinate for legend
 export const getLegendX = ({
-  chartWidth,
+  chartType,
+  ...rest
+}: ChartLegendInterface) => {
+  if (chartType === 'pie') {
+    return getPieLegendX(rest);
+  }
+  return getChartLegendX(rest);
+}
+
+// Returns y coordinate for legend
+export const getLegendY = ({
+  chartType,
+  ...rest
+}: ChartLegendInterface) => {
+  if (chartType === 'pie') {
```

Comment:
(nit) use switch for consistency with the rest of the functions

```js
switch (chartType) {
  case 'pie':
     return getPieLegendY(rest);
  case 'bullet':
    return getBulletLegendY(rest); 
  default:
    return getChartLegendY(rest);
}
```

Reply:
ok